roles/power_aix_vioshc/vars/main.yml

```diff
@@ -0,0 +1,3 @@
+# Copyright (c) IBM Corporation 2020
+
+vioshc_opt_free_size: 500
```

Comment:
might be better if this was in `defaults/main.yml` instead of `vars/main.yml`

Reply:
Updated in latest commit.